src/test/java/com/google/api/codegen/testsrc/libraryproto/library_gapic.yaml

```diff
@@ -577,8 +577,8 @@ interfaces:
       parameters:
         defaults:
         - name=BASIC
-        - comment.comment=comment_file
-        - image=image_file.jpg
+        - comment.comment = comment_file
+        - image="image_file.jpg"
```

Comment:
Could you change this to `image= "image_file.jpg` (space before the quoted string) to make sure that works? Testing that gives me more confidence that it will work without the space than vice-versa. Or you could include both in configs.

Reply:
you mean `image= "image_file.jpg"` right?

Reply:
yep